package.json

```diff
@@ -139,21 +139,21 @@
     "@segment/analytics-next": "1.31.1",
     "@sentry/react": "6.16.1",
     "@sentry/tracing": "6.16.1",
-    "@stacks/auth": "5.0.1",
+    "@stacks/auth": "5.0.3-pr.ea8bc25.0",
     "@stacks/blockchain-api-client": "4.0.1",
-    "@stacks/common": "5.0.0",
-    "@stacks/connect": "7.0.0",
+    "@stacks/common": "5.0.3-pr.ea8bc25.0",
+    "@stacks/connect": "5.0.6-alpha.5071c1c.0",
```

Comment:
What's the need for using a lower version of this package?

Reply:
That is a version build on top of 7.0.0. It contains changes that have not yet been released as a proper version